fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,44 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on label `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    the event corresponding to `y=1`.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to `y = 1` in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, which is just `label=1`
+    (1 unique value), multiplied by two (for the Lagrange multipliers
+    for positive and negative constraints).
```

Comment:
The above sounds still a bit confusing. How about the following re-phrase of lines 170 and 171 to clarify things a bit:

```
    ...
    the number of unique values of the sensitive feature, multiplied by
    the number of unique non-NaN values of the constructed `event` array, which contains only `label=1`
    ...
```